core/src/main/java/org/polypheny/db/algebra/core/document/DocumentTransformer.java

```diff
@@ -39,7 +39,9 @@ public DocumentTransformer( AlgOptCluster cluster, List<AlgNode> inputs, AlgTrai
 
     @Override
     public String algCompareString() {
-        return "$" + outModelTrait + "$" + inModelTrait + "$" + getInput( 0 ).algCompareString();
+        return outModelTrait + "$"
+                + inModelTrait + "$"
+                + getInput( 0 ).algCompareString() + "&";
```

Comment:
Is it safe to only consider the first input?

Reply:
Changed.